Sources/Commands/SwiftPackageTool.swift

```diff
@@ -190,22 +190,25 @@ extension SwiftPackageTool {
         }
     }
 
-    struct Init: SwiftCommand {
-        static let configuration = CommandConfiguration(
+    public struct Init: SwiftCommand {
+        public static let configuration = CommandConfiguration(
             abstract: "Initialize a new package")
 
-        @OptionGroup()
+        @OptionGroup(_hiddenFromHelp: true)
         var swiftOptions: SwiftToolOptions
         
-        @Option(name: .customLong("type"))
+        @Option(name: .customLong("type"), help: "Package type: empty | library | executable | system-module | manifest")
         var initMode: InitPackage.PackageType = .library
         
         @Option(name: .customLong("name"), help: "Provide custom package name")
         var packageName: String?
-
+        
+        public init() {}
+        
         func run(_ swiftTool: SwiftTool) throws {
-            // FIXME: Error handling.
-            let cwd = localFileSystem.currentWorkingDirectory!
+            guard let cwd = localFileSystem.currentWorkingDirectory else {
+                throw StringError("Could not find the current working directory")
```

Comment:
throw InternalError in such case, as it would assert in debug mode which is what we want in this failure case